app/src/main/kotlin/cz/covid19cz/erouska/ui/login/LoginVM.kt

```diff
@@ -184,23 +181,49 @@ class LoginVM(
         }
     }
 
-    private fun handleError(e: Exception) {
-        L.e(e)
-        if (e is FirebaseAuthInvalidCredentialsException) {
-            L.d("Error code: ${e.errorCode}")
+    private fun handleError(exception: Exception) {
+        L.e(exception)
+        if (exception is FirebaseAuthInvalidCredentialsException) {
+            L.d("Error code: ${exception.errorCode}")
         }
-        if (e is FirebaseAuthInvalidCredentialsException && e.errorCode == "ERROR_INVALID_PHONE_NUMBER") {
-            mutableState.postValue(EnterPhoneNumber(true))
-        } else if (e is FirebaseAuthInvalidCredentialsException && e.errorCode == "ERROR_INVALID_VERIFICATION_CODE" || e is FirebaseAuthInvalidCredentialsException) {
```

Comment:
Byl tady ten OR spravne? To prece nemohlo nikdy dojet do tech dalsich if else casu dolu, ne?

Reply:
&& má prioritu před || ne? Máme spíš opačný problém - moc často to skočilo do else větve a nevíme co to je za chybu

Reply:
Prave..proto ti vsechny `FirebaseAuthInvalidCredentialsException` spadnou do tohodle casu